x-pack/plugins/file_data_visualizer/public/application/components/results_links/results_links.tsx

```diff
@@ -33,6 +26,17 @@ interface Props {
   showFilebeatFlyout(): void;
 }
 
+interface GlobalState {
+  time?: {
```

Comment:
How about using `TimeRange` type, src/plugins/data/public, here instead of redefining it?